src/main.scss

```diff
@@ -67,48 +132,713 @@ header {
 				}
 			}
 		}
-		@media only screen and (max-width: 480px) {
-				display: none;
-		}
 	}
 }
 
-        // nav 2 //
-.nav-2 {
+// nav 2 //
+.nav-2 {	
 	background-color: $nav-2-color;
-	.left-buttons{
-		.button-1{
-			img{
-				height: 32px;
+	height: 100px;
+	@media (max-width: $screen-md-min){
+		height: 210px;
+	}
+	@media (max-width: $screen-xs-min){
+		height: 70px;
+	}
+	.container {
+		height: 100%;
+		.l-side-gray-nav {
+			padding-left: 0;
+			@media (max-width: $screen-xs-min) { 
+				padding: 0
 			}
+			
+				.left-buttons {
+					padding-left: 0;
+					@media (max-width: $screen-xs-min) { 
+						width: 100%;
+						padding: 0
+					}
+					.button {
+						direction: rtl;
+						height: 100%;
+						width: 164px;
+						@media (max-width: $screen-xs-min) { 
+							background-color: $nav-2-color;
+							width: 100%;
+						}
+
+						img {
+							height: 32px;
+							@media (max-width: $screen-xs-min){
+								margin: 0;
+							}
+						}
+				
+						p {
+							font-family: $main-font;
+							color: $light-blue-nav;
+							font-size: 17px;
+							margin: 0;
+							text-align: right; 
+							line-height: 21px;
+							@media (max-width: $screen-xs-min) { 
+								font-size: 10px;
+								text-align: center;
+								line-height: 15px;
+							}
+						}
+					}
+
+					.button-3 {
+						padding: 0;
+						height: 100%;
+						background-color: $nav-2-color;
+
+						img {
+							height: 60px;
+							width: 60px;
+							@media (max-width: $screen-xs-min) { 
+							width: 20px;
+							height: 18px;
+							}
+						}
+					}
+
+					.button-1 {
+						margin-right: 3px;
+						@media (max-width: $screen-xs-min) { 
+							margin-right: 0;
+							padding: 0 18px 0 18px;
+						}
+						img {
+							height: 32px;
+							@media (max-width: $screen-xs-min) { 
+								width: 11.8px;
```

Comment:
> `width: 11.8px;`

This doesn't mean anything. You can't give fractions in `px`, it's either `11px` or `12px`